zebra/zapi_msg.c

```diff
@@ -818,6 +826,19 @@ void zsend_rule_notify_owner(const struct zebra_dplane_ctx *ctx,
 	zserv_send_message(client, s);
 }
 
+static void zread_route_notify_request(ZAPI_HANDLER_ARGS)
+{
+	uint8_t notify;
+
+	STREAM_GETC(msg, notify);
+	client->notify_owner = notify;
+
+	if (IS_ZEBRA_DEBUG_RECV)
+		zlog_debug("notify %d", notify);
```

Comment:
We can do better for this log message. This is basically useless unless you wrote the code.